Jellyfin.Server/Migrations/Routines/FixAudioData.cs

```diff
@@ -80,7 +80,7 @@ public void Perform()
                 {
                     IncludeItemTypes = [BaseItemKind.Audio],
                     StartIndex = startIndex,
-                    Limit = 100,
+                    Limit = 5000,
```

Comment:
Two things:
1. startIndex must be updated as well and it should be updated with results.Length
2. Might be nice with some logging on execution time on the DB backup so we can see if that is the issue